launcher/modplatform/legacy_ftb/PackInstallTask.cpp

```diff
@@ -65,6 +65,7 @@ void PackInstallTask::executeTask()
 void PackInstallTask::downloadPack()
 {
     setStatus(tr("Downloading zip for %1").arg(m_pack.name));
+    setProgress(1, 4);
```

Comment:
The only line that fixes anything? :P

Reply:
Nope, the progress signal was weird so I also yanked it out. 
Regarding the format, I presumed that it would be ok to have it as this is a small PR.
If needed I can remove the format

Reply:
> If needed I can remove the format

maybe you can just split it up into two commits to make it easier to separate formatting and actual changes

Reply:
> maybe you can just split it up into two commits to make it easier to separate formatting and actual changes

Thanks for the suggestion

Reply:
so uhhh, this just sets it to 25%? 1/4th

Reply:
yeah but now has a sub-task that progresses normaly

Reply:
thats fine I suppose ^^